src/app/[lang]/(user)/items/[id]/page.tsx

```diff
@@ -0,0 +1,18 @@
+import ProductItem from "@/components/product-item";
+import { productsInStock } from "@/domains/product/productsInStock";
+
+export default function Item({ params }: { params: { id: string } }) {
```

Comment:
I suggest to add `Page` suffix to page-components.

`export default function ItemPage`